logbook-core/src/main/java/org/zalando/logbook/CompositeSink.java

```diff
@@ -0,0 +1,34 @@
+package org.zalando.logbook;
+
+import lombok.AllArgsConstructor;
+
+import java.util.Collection;
+
+import static org.zalando.fauxpas.FauxPas.throwingConsumer;
+
+@AllArgsConstructor
+public final class CompositeSink implements Sink {
+
+    private final Collection<Sink> sinks;
+
+    @Override
+    public boolean isActive() {
+        return sinks.stream().anyMatch(Sink::isActive);
+    }
+
+    @Override
+    public void write(final Precorrelation precorrelation, final HttpRequest request) {
+        sinks.forEach(throwingConsumer(sink -> sink.write(precorrelation, request)));
```

Comment:
Should these be `sinks.filter(Sink::isActive).forEach`?

Reply:
Yes... :man_facepalming: :

Reply:
Fixed